### DISCUSSION TITLE
MQTT icon grey

### DISCUSSION THREAD

**Discussion by Cartoon-Dude** [UtilitechAS/amsreader-firmware#696] [General]
Hello
There is info in the FAQ about the MQTT icon green, yellow, or red.
But what does grey means, as seen in the screenshot?
![Capture d’écran du 2023-12-19 21-46-59](https://github.com/UtilitechAS/amsreader-firmware/assets/133660601/d36d6692-6fec-4ae3-a8f3-a7d7727bfcbe)

**Comment by gskjold** [comment]
Hi, grey just means it is not configured. But yes, we should include that in wiki
